changelog.json

```diff
@@ -1,5 +1,6 @@
 {
   "releases": {
+    "3.4.16-beta2": ["[Removed] Remove support for macOS 10.15 - #19918"],
```

Comment:
@tidy-dev This is incorrect, right?

Reply:
Ha. Yeah. It needs to be.. something about adding a banner. :P

Reply:
Fixed - https://github.com/desktop/desktop/pull/19934/commits/736f4a9742324d502178aeb7bca9cdf8ecff291b